src/current/cockroachcloud/cloud-api.md

```diff
@@ -380,10 +420,66 @@ Where:
 
   - `{cluster_id}` is the unique ID of this cluster.
   {{site.data.alerts.callout_info}}
-  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
+  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
```

Comment:
This is used so many times in this page that it is worth assigning it to a Liquid variable at the top and then using that everywhere. It will be less brittle to add it to the various different contexts and get the indentation right.

Reply:
Adding this to follow-up review